controls/bsi_app_4_4.yml

```diff
@@ -162,35 +162,46 @@ controls:
     levels:
     - standard
     description: >-
-      If an initialisation (e.g. of an application) takes place in a pod at start-up, this SHOULD take
-      place in a separate Init container. It SHOULD be ensured that the initialisation terminates all
-      processes that are already running. Kubernetes SHOULD ONLY start the other containers if
-      the initialisation is successful.
+      If an initialisation (e.g. of an application) takes place in a pod at start-up, this SHOULD take place in a separate Init container. It SHOULD be ensured that the initialisation terminates all processes that are already running. Kubernetes SHOULD ONLY start the other containers if the initialisation is successful.

```

Comment:
@sluetze I see this paragraph now matches exactly the text on BSI pdf.

Is this intentional? I'd rather avoid very long lines to ease readability.

Reply:
Yes this is intentional and true for all controls in bsi_app_4_4 and bsi_sys_1_6
We created the skeleton of the control file by using the BSI pdf texts as description. We did so for two reasons. First we did not want to spread too much information over too many docs to avoid the necessity to switch between docs and versions all the time. Secondly this way we do not have to "invent" a new description which might lead to misunderstandings or be "slightly off" in comparison to what the bsi pdf writes.

Reply:
I definitely support having descriptions identical to the pdf.
What I'm not much a fan of are the very long lines, if you are okay, and it is possible, wrap them around at 99 characters.